nexus_constructor/model/transformation.py

```diff
@@ -0,0 +1,72 @@
+import attr
+from PySide2.Qt3DCore import Qt3DCore
+from PySide2.QtGui import QVector3D, QMatrix4x4
+import numpy as np
+
+from nexus_constructor.model.group import Group
+from nexus_constructor.model.helpers import get_item, set_item
+from nexus_constructor.common_attrs import CommonAttrs
+from nexus_constructor.transformation_types import TransformationType
+
+
+class TransformationGroup(Group):
```

Comment:
Is this class needed? I honestly do not remember enough of the NeXus documentation to be able to say.